src/components/logo/logo.tsx

```diff
@@ -0,0 +1,64 @@
+import {
+  Heading,
+  VStack,
+  HStack,
+  Logo,
+  Link,
+} from "@nypl/design-system-react-components";
+import React from "react";
+import appConfig from "appConfig";
+
+const DCLogo = () => {
+  return (
+    <>
+      <Link isUnderlined={false} href={`${appConfig.DC_URL}`}>
+        <HStack>
+          <Logo
+            pt="2"
+            pb="2"
+            name="nyplLionBlack"
```

Comment:
There's an SVG type for the logo called `digitalCollectionsBlack` https://nypl.github.io/nypl-design-system/reservoir/v2/?path=/story/components-media-icons-logo--with-controls&args=name:digitalCollectionsBlack. Did you try this and it didn't render properly? If possible, try to use this. 

Sorry, that should've been in the ticket. If it does work, you may need to update this so it renders only this `"nyplLionBlack"` logo on mobile and `"digitalCollectionsBlack"` on desktop.

Reply:
I should asked about this earlier– it renders but I couldn't find a way to make it large enough?

Reply:
There's a new `sizeBasedOn` prop for the `Logo` component that might help make it render bigger. But this new prop is in 2.1.2. You could try to override the size (width/height) with the `sx` prop if the new prop doesn't work.

Reply:
`sx` isn't overriding either unfortunately

Reply:
If you install DS version 2.1.2, you can add this to the component
```
height="80px"
sizeBasedOn="height"
```

While this does help, it's not accurate so you would need to adjust the height value to get it closer to the figma file.

Reply:
Awesome, sorry for some reason didn't realize from your above comment that updating to 2.1.2 was a possible thing to do